bankaccount.go

```diff
@@ -141,6 +141,7 @@ func (p *BankAccountListParams) AppendTo(body *form.Values, keyParts []string) {
 
 // BankAccount represents a Stripe bank account.
 type BankAccount struct {
+	Account            string                       `json:"account"`
```

Comment:
This should be `Account *Account `json:"account"` instead. This property is expandable in the API and so we need to build support for the expanded version even though it can not be expanded in your case because it comes from an event

Reply:
All set! (and tested working for me)